src/index.js

```diff
@@ -185,12 +185,26 @@ function unmountComponentAtNode(container) {
 const ARR = [];
 
 // This API is completely unnecessary for Preact, so it's basically passthrough.
+
 let Children = {
 	map(children, fn, ctx) {
 		if (children == null) return null;
 		children = Children.toArray(children);
 		if (ctx && ctx!==children) fn = fn.bind(ctx);
-		return children.map(fn);
```

Comment:
why not `return children.map(fn).filter(item => item !== null);`

Reply:
when the children has `true` or `false`, React will ignore them, see the demo

```js
// demo
let instance = (
    <div>
      {'123'}
      {true}
      {false}
    </div>
);
class Hello extends React.Component {
  render() {
    console.info(instance.props.children); //  ["123", true, false]
    const ChildrenMap = React.Children.map(instance.props.children, item => item);
    console.info(ChildrenMap); // ["123"]
    return <div>
     {ChildrenMap}
    </div>
  }
}
ReactDOM.render(<Hello/>, document.getElementById('test'));    
```

Reply:
ok, i see we need to skip boolean and null, undefined

Reply:
Doesn't it only skip rendering?

Reply:
Yeah, react does much more in `React.Children.Map`, but I think skip rendering is the the main difference between `react-compat` and react.  The main purpose of this pull request is to improve compatibility.